### DISCUSSION TITLE
Upcoming releases, current priorities, big plans

### DISCUSSION THREAD

**Discussion by cabauman** [MichalLytek/type-graphql#927] [General]
Hey Michal! I'm new to this community, so I just wanted to break the ice. I've only been using TypeGraphQL for about 6 months (and Typescript for about a year, coming from a C# background), but am already a big fan of your work on this library.

I'm interested in learning more about your overall plans for the project, what your main focus is at the moment, and if you have a roadmap or any plans for upcoming releases. Looking forward to some of the new features you've been adding since the last release. I hope to eventually start contributing, as well.

**Comment by carlocorradini** [comment]
Hey @cabauman, have you seen GitHub's [Projects](https://github.com/MichalLytek/type-graphql/projects/2)?

Here you can find a detailed *list*: ```Primary goals | To do | In progress | Done```.

**Reply by cabauman** [reply]
Ah I hadn't seen it for this particular repository. Thanks for pointing that out.

**Reply by carlocorradini** [reply]
@cabauman Can we close this?

**Reply by carlocorradini** [reply]
In addition there are the [Blog](https://typegraphql.com/blog) and [Twitter](https://twitter.com/typegraphql)

**Reply by cabauman** [reply]
Sure, we can close it. Thanks again.

**Reply by carlocorradini** [reply]
> Sure, we can close it. Thanks again.

Ok, Accept the answer as *answered*. Thanks!

**Reply by cabauman** [reply]
Correct me if I'm mistaken, but I don't think it's possible for me to mark a response as _answered_ because this discussion is classified as _general_ rather than a _question_. I was mainly looking for community interaction when I created this thread.
